.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug report
+description: File a bug report to help us improve
+labels: ["bug", "needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ####
+        Before starting a bug report, please take a look into the [Troubleshooting Documentation](https://docs.edgeless.systems/contrast/troubleshooting) and revise the existing issue.
+        Further please ensure that the provided CLI logs are configured with the `--log-level` command-line flag set to `debug`.
+
+        Thanks for taking the time to fill out this bug report!
+
+  - type: input
+    id: version
+    attributes:
+      label: Contrast Version
+      description: "In which version do you encounter this bug? (Output of `contrast version`) "
```

Comment:
There is no `contrast version`, only `contrast --version`. Also that will print a lot, I don't think we want all of that in the field, right?

Reply:
Consider adding instructions for reporting unreleased bugs.